### DISCUSSION TITLE
How to have my Stencil Checkout link back to my Catalyst Store

### DISCUSSION THREAD

**Discussion by danielalexvega** [bigcommerce/catalyst#996] [Q&A]
🔥 🔥 🔥 🔥 🔥 

Hello, I have a Catalyst store launched on Vercel, and everything so far is working. When the customer adds something to the cart, and checks out, they are directed to the Stencil checkout. 

I've customized the Stencil checkout so that the logo in the header links back to the Catalyst site (and now the Stencil store), but on the Order Confirmation page, there's a script 

`{{{ checkout.order_confirmation_content }}}`

And I don't know how to get that to link back to my Catalyst site. 

Here is my Stencil site. https://cape-ann-fresh-catch-demo.mybigcommerce.com/
Here is my Catalyst site. https://twenty-four-hour-corner-store.vercel.app/

Thanks

**Comment by bookernath** [comment]
Hi @danielalexvega , you just need to set up your Routes: https://developer.bigcommerce.com/docs/rest-management/sites/site-routes#update-a-sites-routes

Here's an example payload we use for base catalyst:

```
[
  {
    "type": "home",
    "matching": "*",
    "route": "/"
  },
  {
    "type": "cart",
    "matching": "*",
    "route": "/cart/"
  },
  {
    "type": "login",
    "matching": "*",
    "route": "/login/"
  },
  {
    "type": "search",
    "matching": "*",
    "route": "/search/"
  },
  {
    "type": "account",
    "matching": "*",
    "route": "/account/"
  }
]
```
